ccl/backupccl: TestBackupRestoreDataDriven failed

ccl/backupccl.TestBackupRestoreDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4488322&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4488322&tab=artifacts#/) on master @ [573b3b05458667308bafeb9db1611832dca3025f](https://github.com/cockroachdb/cockroach/commits/573b3b05458667308bafeb9db1611832dca3025f):


```
=== RUN   TestBackupRestoreDataDriven
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/2b7e992b6b49d0a96a6be6a67faab5b1/logTestBackupRestoreDataDriven369607015
    test_log_scope.go:80: use -show-logs to present logs inline
    backup_test.go:274: Mock HTTP Storage "http://127.0.0.1:44209"
=== CONT  TestBackupRestoreDataDriven
    backup_test.go:378: -- test log scope end --
--- FAIL: TestBackupRestoreDataDriven (68.43s)
=== RUN   TestBackupRestoreDataDriven/column-families
    backup_test.go:282: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3372/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:1:
        new-server [2 args]
        <no input to command>
        ----
    backup_test.go:282: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3372/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:4:
        exec-sql [0 args]
        CREATE DATABASE orig;
        USE orig;
        CREATE TABLE cfs (a INT PRIMARY KEY, b STRING, c STRING, d STRING, FAMILY (b), FAMILY (c));
        INSERT INTO cfs SELECT x, repeat('abc', 100), repeat('abc', 100) FROM generate_series(0, 3) AS x;
        ALTER TABLE cfs SPLIT AT SELECT a FROM cfs;
        -- Split the output files very small to catch output SSTs mid-row.
        SET CLUSTER SETTING bulkio.backup.file_size = '1';
        SET CLUSTER SETTING kv.bulk_sst.target_size = '1';
        SET CLUSTER SETTING bulkio.backup.merge_file_buffer_size = '1MiB';
        BACKUP cfs TO 'nodelocal://1/foo';
        CREATE DATABASE r1;
        RESTORE cfs FROM 'nodelocal://1/foo' WITH into_db='r1';
        ----
    backup_test.go:282: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3372/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:19:
        query-sql [0 args]
        SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE orig.cfs];
        ----
        2
    backup_test.go:282: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3372/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:28: SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE r1.cfs];
        expected:
        2
        
        found:
        <nil>
    --- FAIL: TestBackupRestoreDataDriven/column-families (3.27s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
/cc @cockroachdb/bulk-io
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-13506

ccl/backupccl.TestBackupRestoreDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4496855&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4496855&tab=artifacts#/) on master @ [dbdbb93978a9ad564e07085c15807f415ccaa00a](https://github.com/cockroachdb/cockroach/commits/dbdbb93978a9ad564e07085c15807f415ccaa00a):


```
=== RUN   TestBackupRestoreDataDriven
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/2b7e992b6b49d0a96a6be6a67faab5b1/logTestBackupRestoreDataDriven517848907
    test_log_scope.go:80: use -show-logs to present logs inline
    backup_test.go:274: Mock HTTP Storage "http://127.0.0.1:42685"
=== CONT  TestBackupRestoreDataDriven
    backup_test.go:378: -- test log scope end --
--- FAIL: TestBackupRestoreDataDriven (67.55s)
=== RUN   TestBackupRestoreDataDriven/column-families
    backup_test.go:282: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3557/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:1:
        new-server [2 args]
        <no input to command>
        ----
    backup_test.go:282: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3557/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:4:
        exec-sql [0 args]
        CREATE DATABASE orig;
        USE orig;
        CREATE TABLE cfs (a INT PRIMARY KEY, b STRING, c STRING, d STRING, FAMILY (b), FAMILY (c));
        INSERT INTO cfs SELECT x, repeat('abc', 100), repeat('abc', 100) FROM generate_series(0, 3) AS x;
        ALTER TABLE cfs SPLIT AT SELECT a FROM cfs;
        -- Split the output files very small to catch output SSTs mid-row.
        SET CLUSTER SETTING bulkio.backup.file_size = '1';
        SET CLUSTER SETTING kv.bulk_sst.target_size = '1';
        SET CLUSTER SETTING bulkio.backup.merge_file_buffer_size = '1MiB';
        BACKUP cfs TO 'nodelocal://1/foo';
        CREATE DATABASE r1;
        RESTORE cfs FROM 'nodelocal://1/foo' WITH into_db='r1';
        ----
    backup_test.go:282: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3557/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:19:
        query-sql [0 args]
        SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE orig.cfs];
        ----
        2
    backup_test.go:282: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3557/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:28: SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE r1.cfs];
        expected:
        2
        
        found:
        <nil>
    --- FAIL: TestBackupRestoreDataDriven/column-families (3.29s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupRestoreDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4505613&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4505613&tab=artifacts#/) on master @ [0e1edf3ad9da29b1d3bb28fcfc0f293d700e3cb9](https://github.com/cockroachdb/cockroach/commits/0e1edf3ad9da29b1d3bb28fcfc0f293d700e3cb9):


```
=== RUN   TestBackupRestoreDataDriven
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/2b7e992b6b49d0a96a6be6a67faab5b1/logTestBackupRestoreDataDriven679541443
    test_log_scope.go:80: use -show-logs to present logs inline
    backup_test.go:274: Mock HTTP Storage "http://127.0.0.1:35261"
=== CONT  TestBackupRestoreDataDriven
    backup_test.go:378: -- test log scope end --
--- FAIL: TestBackupRestoreDataDriven (64.11s)
=== RUN   TestBackupRestoreDataDriven/column-families
    backup_test.go:282: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3181/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:1:
        new-server [2 args]
        <no input to command>
        ----
    backup_test.go:282: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3181/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:4:
        exec-sql [0 args]
        CREATE DATABASE orig;
        USE orig;
        CREATE TABLE cfs (a INT PRIMARY KEY, b STRING, c STRING, d STRING, FAMILY (b), FAMILY (c));
        INSERT INTO cfs SELECT x, repeat('abc', 100), repeat('abc', 100) FROM generate_series(0, 3) AS x;
        ALTER TABLE cfs SPLIT AT SELECT a FROM cfs;
        -- Split the output files very small to catch output SSTs mid-row.
        SET CLUSTER SETTING bulkio.backup.file_size = '1';
        SET CLUSTER SETTING kv.bulk_sst.target_size = '1';
        SET CLUSTER SETTING bulkio.backup.merge_file_buffer_size = '1MiB';
        BACKUP cfs TO 'nodelocal://1/foo';
        CREATE DATABASE r1;
        RESTORE cfs FROM 'nodelocal://1/foo' WITH into_db='r1';
        ----
    backup_test.go:282: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3181/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:19:
        query-sql [0 args]
        SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE orig.cfs];
        ----
        2
    backup_test.go:282: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3181/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:28: SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE r1.cfs];
        expected:
        2
        
        found:
        <nil>
    --- FAIL: TestBackupRestoreDataDriven/column-families (3.70s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupRestoreDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4510299&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4510299&tab=artifacts#/) on master @ [71eb44ffb1574350cf4a36143ed423f6ac0ca02b](https://github.com/cockroachdb/cockroach/commits/71eb44ffb1574350cf4a36143ed423f6ac0ca02b):


```
=== RUN   TestBackupRestoreDataDriven
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/2b7e992b6b49d0a96a6be6a67faab5b1/logTestBackupRestoreDataDriven3098346139
    test_log_scope.go:80: use -show-logs to present logs inline
    backup_test.go:274: Mock HTTP Storage "http://127.0.0.1:44909"
=== CONT  TestBackupRestoreDataDriven
    backup_test.go:378: -- test log scope end --
--- FAIL: TestBackupRestoreDataDriven (66.57s)
=== RUN   TestBackupRestoreDataDriven/column-families
    backup_test.go:282: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2868/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:1:
        new-server [2 args]
        <no input to command>
        ----
    backup_test.go:282: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2868/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:4:
        exec-sql [0 args]
        CREATE DATABASE orig;
        USE orig;
        CREATE TABLE cfs (a INT PRIMARY KEY, b STRING, c STRING, d STRING, FAMILY (b), FAMILY (c));
        INSERT INTO cfs SELECT x, repeat('abc', 100), repeat('abc', 100) FROM generate_series(0, 3) AS x;
        ALTER TABLE cfs SPLIT AT SELECT a FROM cfs;
        -- Split the output files very small to catch output SSTs mid-row.
        SET CLUSTER SETTING bulkio.backup.file_size = '1';
        SET CLUSTER SETTING kv.bulk_sst.target_size = '1';
        SET CLUSTER SETTING bulkio.backup.merge_file_buffer_size = '1MiB';
        BACKUP cfs TO 'nodelocal://1/foo';
        CREATE DATABASE r1;
        RESTORE cfs FROM 'nodelocal://1/foo' WITH into_db='r1';
        ----
    backup_test.go:282: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2868/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:19:
        query-sql [0 args]
        SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE orig.cfs];
        ----
        2
    backup_test.go:282: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2868/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:28: SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE r1.cfs];
        expected:
        2
        
        found:
        <nil>
    --- FAIL: TestBackupRestoreDataDriven/column-families (4.24s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupRestoreDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4531512&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4531512&tab=artifacts#/) on master @ [e7e596ede9d3da877e313a3d39ab498a5cb7d27b](https://github.com/cockroachdb/cockroach/commits/e7e596ede9d3da877e313a3d39ab498a5cb7d27b):


```
=== RUN   TestBackupRestoreDataDriven
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/2b7e992b6b49d0a96a6be6a67faab5b1/logTestBackupRestoreDataDriven2850124086
    test_log_scope.go:80: use -show-logs to present logs inline
    backup_test.go:274: Mock HTTP Storage "http://127.0.0.1:36807"
=== CONT  TestBackupRestoreDataDriven
    backup_test.go:378: -- test log scope end --
--- FAIL: TestBackupRestoreDataDriven (68.37s)
=== RUN   TestBackupRestoreDataDriven/column-families
    backup_test.go:282: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3161/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:1:
        new-server [2 args]
        <no input to command>
        ----
    backup_test.go:282: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3161/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:4:
        exec-sql [0 args]
        CREATE DATABASE orig;
        USE orig;
        CREATE TABLE cfs (a INT PRIMARY KEY, b STRING, c STRING, d STRING, FAMILY (b), FAMILY (c));
        INSERT INTO cfs SELECT x, repeat('abc', 100), repeat('abc', 100) FROM generate_series(0, 3) AS x;
        ALTER TABLE cfs SPLIT AT SELECT a FROM cfs;
        -- Split the output files very small to catch output SSTs mid-row.
        SET CLUSTER SETTING bulkio.backup.file_size = '1';
        SET CLUSTER SETTING kv.bulk_sst.target_size = '1';
        SET CLUSTER SETTING bulkio.backup.merge_file_buffer_size = '1MiB';
        BACKUP cfs TO 'nodelocal://1/foo';
        CREATE DATABASE r1;
        RESTORE cfs FROM 'nodelocal://1/foo' WITH into_db='r1';
        ----
    backup_test.go:282: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3161/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:19:
        query-sql [0 args]
        SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE orig.cfs];
        ----
        2
    backup_test.go:282: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3161/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:28: SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE r1.cfs];
        expected:
        2
        
        found:
        <nil>
    --- FAIL: TestBackupRestoreDataDriven/column-families (4.00s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupRestoreDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4549968&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4549968&tab=artifacts#/) on master @ [0069e980d756ca580167eb321ac6b348b35313f0](https://github.com/cockroachdb/cockroach/commits/0069e980d756ca580167eb321ac6b348b35313f0):


```
=== RUN   TestBackupRestoreDataDriven
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/2b7e992b6b49d0a96a6be6a67faab5b1/logTestBackupRestoreDataDriven2160304396
    test_log_scope.go:80: use -show-logs to present logs inline
    backup_test.go:275: Mock HTTP Storage "http://127.0.0.1:43735"
=== CONT  TestBackupRestoreDataDriven
    backup_test.go:379: -- test log scope end --
--- FAIL: TestBackupRestoreDataDriven (64.04s)
=== RUN   TestBackupRestoreDataDriven/column-families
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:1:
        new-server [2 args]
        <no input to command>
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:4:
        exec-sql [0 args]
        CREATE DATABASE orig;
        USE orig;
        CREATE TABLE cfs (a INT PRIMARY KEY, b STRING, c STRING, d STRING, FAMILY (b), FAMILY (c));
        INSERT INTO cfs SELECT x, repeat('abc', 100), repeat('abc', 100) FROM generate_series(0, 3) AS x;
        ALTER TABLE cfs SPLIT AT SELECT a FROM cfs;
        -- Split the output files very small to catch output SSTs mid-row.
        SET CLUSTER SETTING bulkio.backup.file_size = '1';
        SET CLUSTER SETTING kv.bulk_sst.target_size = '1';
        SET CLUSTER SETTING bulkio.backup.merge_file_buffer_size = '1MiB';
        BACKUP cfs TO 'nodelocal://1/foo';
        CREATE DATABASE r1;
        RESTORE cfs FROM 'nodelocal://1/foo' WITH into_db='r1';
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:19:
        query-sql [0 args]
        SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE orig.cfs];
        ----
        2
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:28: SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE r1.cfs];
        expected:
        2
        
        found:
        <nil>
    --- FAIL: TestBackupRestoreDataDriven/column-families (3.10s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupRestoreDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4558076&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4558076&tab=artifacts#/) on master @ [270eb7b3ab88fc47ba86d87f4e2113e560f047a8](https://github.com/cockroachdb/cockroach/commits/270eb7b3ab88fc47ba86d87f4e2113e560f047a8):


```
=== RUN   TestBackupRestoreDataDriven
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/2b7e992b6b49d0a96a6be6a67faab5b1/logTestBackupRestoreDataDriven3289378018
    test_log_scope.go:80: use -show-logs to present logs inline
    backup_test.go:275: Mock HTTP Storage "http://127.0.0.1:39605"
=== CONT  TestBackupRestoreDataDriven
    backup_test.go:379: -- test log scope end --
--- FAIL: TestBackupRestoreDataDriven (67.56s)
=== RUN   TestBackupRestoreDataDriven/column-families
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:1:
        new-server [2 args]
        <no input to command>
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:4:
        exec-sql [0 args]
        CREATE DATABASE orig;
        USE orig;
        CREATE TABLE cfs (a INT PRIMARY KEY, b STRING, c STRING, d STRING, FAMILY (b), FAMILY (c));
        INSERT INTO cfs SELECT x, repeat('abc', 100), repeat('abc', 100) FROM generate_series(0, 3) AS x;
        ALTER TABLE cfs SPLIT AT SELECT a FROM cfs;
        -- Split the output files very small to catch output SSTs mid-row.
        SET CLUSTER SETTING bulkio.backup.file_size = '1';
        SET CLUSTER SETTING kv.bulk_sst.target_size = '1';
        SET CLUSTER SETTING bulkio.backup.merge_file_buffer_size = '1MiB';
        BACKUP cfs TO 'nodelocal://1/foo';
        CREATE DATABASE r1;
        RESTORE cfs FROM 'nodelocal://1/foo' WITH into_db='r1';
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:19:
        query-sql [0 args]
        SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE orig.cfs];
        ----
        2
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:28: SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE r1.cfs];
        expected:
        2
        
        found:
        <nil>
    --- FAIL: TestBackupRestoreDataDriven/column-families (3.41s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupRestoreDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4561723&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4561723&tab=artifacts#/) on master @ [dc5432d65b9c38ce1ce5832e3c49d345b66145dc](https://github.com/cockroachdb/cockroach/commits/dc5432d65b9c38ce1ce5832e3c49d345b66145dc):


```
=== RUN   TestBackupRestoreDataDriven
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/2b7e992b6b49d0a96a6be6a67faab5b1/logTestBackupRestoreDataDriven3154294278
    test_log_scope.go:80: use -show-logs to present logs inline
    backup_test.go:275: Mock HTTP Storage "http://127.0.0.1:37147"
=== CONT  TestBackupRestoreDataDriven
    backup_test.go:379: -- test log scope end --
--- FAIL: TestBackupRestoreDataDriven (65.39s)
=== RUN   TestBackupRestoreDataDriven/column-families
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:1:
        new-server [2 args]
        <no input to command>
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:4:
        exec-sql [0 args]
        CREATE DATABASE orig;
        USE orig;
        CREATE TABLE cfs (a INT PRIMARY KEY, b STRING, c STRING, d STRING, FAMILY (b), FAMILY (c));
        INSERT INTO cfs SELECT x, repeat('abc', 100), repeat('abc', 100) FROM generate_series(0, 3) AS x;
        ALTER TABLE cfs SPLIT AT SELECT a FROM cfs;
        -- Split the output files very small to catch output SSTs mid-row.
        SET CLUSTER SETTING bulkio.backup.file_size = '1';
        SET CLUSTER SETTING kv.bulk_sst.target_size = '1';
        SET CLUSTER SETTING bulkio.backup.merge_file_buffer_size = '1MiB';
        BACKUP cfs TO 'nodelocal://1/foo';
        CREATE DATABASE r1;
        RESTORE cfs FROM 'nodelocal://1/foo' WITH into_db='r1';
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:19:
        query-sql [0 args]
        SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE orig.cfs];
        ----
        2
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:28: SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE r1.cfs];
        expected:
        2
        
        found:
        <nil>
    --- FAIL: TestBackupRestoreDataDriven/column-families (4.73s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupRestoreDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4565949&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4565949&tab=artifacts#/) on master @ [231fd420c61e39b6ad08c982c496d82cf1910bc5](https://github.com/cockroachdb/cockroach/commits/231fd420c61e39b6ad08c982c496d82cf1910bc5):


```
=== RUN   TestBackupRestoreDataDriven
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/2b7e992b6b49d0a96a6be6a67faab5b1/logTestBackupRestoreDataDriven4010910029
    test_log_scope.go:80: use -show-logs to present logs inline
    backup_test.go:275: Mock HTTP Storage "http://127.0.0.1:33089"
=== CONT  TestBackupRestoreDataDriven
    backup_test.go:379: -- test log scope end --
--- FAIL: TestBackupRestoreDataDriven (69.54s)
=== RUN   TestBackupRestoreDataDriven/column-families
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:1:
        new-server [2 args]
        <no input to command>
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:4:
        exec-sql [0 args]
        CREATE DATABASE orig;
        USE orig;
        CREATE TABLE cfs (a INT PRIMARY KEY, b STRING, c STRING, d STRING, FAMILY (b), FAMILY (c));
        INSERT INTO cfs SELECT x, repeat('abc', 100), repeat('abc', 100) FROM generate_series(0, 3) AS x;
        ALTER TABLE cfs SPLIT AT SELECT a FROM cfs;
        -- Split the output files very small to catch output SSTs mid-row.
        SET CLUSTER SETTING bulkio.backup.file_size = '1';
        SET CLUSTER SETTING kv.bulk_sst.target_size = '1';
        SET CLUSTER SETTING bulkio.backup.merge_file_buffer_size = '1MiB';
        BACKUP cfs TO 'nodelocal://1/foo';
        CREATE DATABASE r1;
        RESTORE cfs FROM 'nodelocal://1/foo' WITH into_db='r1';
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:19:
        query-sql [0 args]
        SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE orig.cfs];
        ----
        2
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:28: SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE r1.cfs];
        expected:
        2
        
        found:
        <nil>
    --- FAIL: TestBackupRestoreDataDriven/column-families (9.38s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupRestoreDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4574827&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4574827&tab=artifacts#/) on master @ [f5fc84fb5707428ae9505c5e3e90cf3f63d465ad](https://github.com/cockroachdb/cockroach/commits/f5fc84fb5707428ae9505c5e3e90cf3f63d465ad):


```
=== RUN   TestBackupRestoreDataDriven
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/2b7e992b6b49d0a96a6be6a67faab5b1/logTestBackupRestoreDataDriven1087918198
    test_log_scope.go:80: use -show-logs to present logs inline
    backup_test.go:275: Mock HTTP Storage "http://127.0.0.1:33477"
=== CONT  TestBackupRestoreDataDriven
    backup_test.go:379: -- test log scope end --
--- FAIL: TestBackupRestoreDataDriven (63.08s)
=== RUN   TestBackupRestoreDataDriven/column-families
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:1:
        new-server [2 args]
        <no input to command>
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:4:
        exec-sql [0 args]
        CREATE DATABASE orig;
        USE orig;
        CREATE TABLE cfs (a INT PRIMARY KEY, b STRING, c STRING, d STRING, FAMILY (b), FAMILY (c));
        INSERT INTO cfs SELECT x, repeat('abc', 100), repeat('abc', 100) FROM generate_series(0, 3) AS x;
        ALTER TABLE cfs SPLIT AT SELECT a FROM cfs;
        -- Split the output files very small to catch output SSTs mid-row.
        SET CLUSTER SETTING bulkio.backup.file_size = '1';
        SET CLUSTER SETTING kv.bulk_sst.target_size = '1';
        SET CLUSTER SETTING bulkio.backup.merge_file_buffer_size = '1MiB';
        BACKUP cfs TO 'nodelocal://1/foo';
        CREATE DATABASE r1;
        RESTORE cfs FROM 'nodelocal://1/foo' WITH into_db='r1';
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:19:
        query-sql [0 args]
        SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE orig.cfs];
        ----
        2
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:28: SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE r1.cfs];
        expected:
        2
        
        found:
        <nil>
    --- FAIL: TestBackupRestoreDataDriven/column-families (3.76s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupRestoreDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4583187&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4583187&tab=artifacts#/) on master @ [2f051c95ebb991e18134d2454807f992b05e4929](https://github.com/cockroachdb/cockroach/commits/2f051c95ebb991e18134d2454807f992b05e4929):


```
        exec-sql [0 args]
        CREATE TABLE cfs (a INT PRIMARY KEY, b STRING, c STRING, d STRING, FAMILY (b), FAMILY (c));
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:16:
        exec-sql [0 args]
        INSERT INTO cfs SELECT x, repeat('abc', 100), repeat('abc', 100) FROM generate_series(0, 3) AS x;
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:20:
        exec-sql [0 args]
        ALTER TABLE cfs SPLIT AT SELECT a FROM cfs;
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:24:
        exec-sql [0 args]
        -- Split the output files very small to catch output SSTs mid-row.
        SET CLUSTER SETTING bulkio.backup.file_size = '1';
        SET CLUSTER SETTING kv.bulk_sst.target_size = '1';
        SET CLUSTER SETTING bulkio.backup.merge_file_buffer_size = '1MiB';
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:31:
        exec-sql [0 args]
        BACKUP cfs TO 'nodelocal://1/foo';
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:35:
        exec-sql [0 args]
        CREATE DATABASE r1;
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:39:
        exec-sql [0 args]
        RESTORE cfs FROM 'nodelocal://1/foo' WITH into_db='r1';
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:43:
        query-sql [0 args]
        SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE orig.cfs];
        ----
        2
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3160/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:52: SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE r1.cfs];
        expected:
        2
        
        found:
        <nil>
    --- FAIL: TestBackupRestoreDataDriven/column-families (11.71s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupRestoreDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4594194&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4594194&tab=artifacts#/) on master @ [8179b6f996149ef9e04eb62f9c95ecc992b9f30e](https://github.com/cockroachdb/cockroach/commits/8179b6f996149ef9e04eb62f9c95ecc992b9f30e):


```
        exec-sql [0 args]
        CREATE TABLE cfs (a INT PRIMARY KEY, b STRING, c STRING, d STRING, FAMILY (b), FAMILY (c));
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:16:
        exec-sql [0 args]
        INSERT INTO cfs SELECT x, repeat('abc', 100), repeat('abc', 100) FROM generate_series(0, 3) AS x;
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:20:
        exec-sql [0 args]
        ALTER TABLE cfs SPLIT AT SELECT a FROM cfs;
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:24:
        exec-sql [0 args]
        -- Split the output files very small to catch output SSTs mid-row.
        SET CLUSTER SETTING bulkio.backup.file_size = '1';
        SET CLUSTER SETTING kv.bulk_sst.target_size = '1';
        SET CLUSTER SETTING bulkio.backup.merge_file_buffer_size = '1MiB';
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:31:
        exec-sql [0 args]
        BACKUP cfs TO 'nodelocal://1/foo';
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:35:
        exec-sql [0 args]
        CREATE DATABASE r1;
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:39:
        exec-sql [0 args]
        RESTORE cfs FROM 'nodelocal://1/foo' WITH into_db='r1';
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:43:
        query-sql [0 args]
        SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE orig.cfs];
        ----
        2
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:52: SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE r1.cfs];
        expected:
        2
        
        found:
        <nil>
    --- FAIL: TestBackupRestoreDataDriven/column-families (3.36s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupRestoreDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4605149&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4605149&tab=artifacts#/) on master @ [9850f2aed2d44e04e08d9f14ff0c98a2bd193ee9](https://github.com/cockroachdb/cockroach/commits/9850f2aed2d44e04e08d9f14ff0c98a2bd193ee9):


```
        exec-sql [0 args]
        CREATE TABLE cfs (a INT PRIMARY KEY, b STRING, c STRING, d STRING, FAMILY (b), FAMILY (c));
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:16:
        exec-sql [0 args]
        INSERT INTO cfs SELECT x, repeat('abc', 100), repeat('abc', 100) FROM generate_series(0, 3) AS x;
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:20:
        exec-sql [0 args]
        ALTER TABLE cfs SPLIT AT SELECT a FROM cfs;
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:24:
        exec-sql [0 args]
        -- Split the output files very small to catch output SSTs mid-row.
        SET CLUSTER SETTING bulkio.backup.file_size = '1';
        SET CLUSTER SETTING kv.bulk_sst.target_size = '1';
        SET CLUSTER SETTING bulkio.backup.merge_file_buffer_size = '1MiB';
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:31:
        exec-sql [0 args]
        BACKUP cfs TO 'nodelocal://1/foo';
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:35:
        exec-sql [0 args]
        CREATE DATABASE r1;
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:39:
        exec-sql [0 args]
        RESTORE cfs FROM 'nodelocal://1/foo' WITH into_db='r1';
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:43:
        query-sql [0 args]
        SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE orig.cfs];
        ----
        2
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:52: SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE r1.cfs];
        expected:
        2
        
        found:
        <nil>
    --- FAIL: TestBackupRestoreDataDriven/column-families (4.36s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #78069 ccl/backupccl: TestBackupRestoreDataDriven failed [C-test-failure O-robot T-bulkio branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupRestoreDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4620046&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4620046&tab=artifacts#/) on master @ [b3525c95b176f7520173e7a6d3e4a93dca86b198](https://github.com/cockroachdb/cockroach/commits/b3525c95b176f7520173e7a6d3e4a93dca86b198):


```
        exec-sql [0 args]
        CREATE TABLE cfs (a INT PRIMARY KEY, b STRING, c STRING, d STRING, FAMILY (b), FAMILY (c));
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:16:
        exec-sql [0 args]
        INSERT INTO cfs SELECT x, repeat('abc', 100), repeat('abc', 100) FROM generate_series(0, 3) AS x;
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:20:
        exec-sql [0 args]
        ALTER TABLE cfs SPLIT AT SELECT a FROM cfs;
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:24:
        exec-sql [0 args]
        -- Split the output files very small to catch output SSTs mid-row.
        SET CLUSTER SETTING bulkio.backup.file_size = '1';
        SET CLUSTER SETTING kv.bulk_sst.target_size = '1';
        SET CLUSTER SETTING bulkio.backup.merge_file_buffer_size = '1MiB';
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:31:
        exec-sql [0 args]
        BACKUP cfs TO 'nodelocal://1/foo';
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:35:
        exec-sql [0 args]
        CREATE DATABASE r1;
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:39:
        exec-sql [0 args]
        RESTORE cfs FROM 'nodelocal://1/foo' WITH into_db='r1';
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:43:
        query-sql [0 args]
        SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE orig.cfs];
        ----
        2
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:52: SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE r1.cfs];
        expected:
        2
        
        found:
        <nil>
    --- FAIL: TestBackupRestoreDataDriven/column-families (2.94s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #78069 ccl/backupccl: TestBackupRestoreDataDriven failed [C-test-failure O-robot T-bulkio branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupRestoreDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4632730&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4632730&tab=artifacts#/) on master @ [5bc7664396123e61f3546d2482b38e4aa0878ed6](https://github.com/cockroachdb/cockroach/commits/5bc7664396123e61f3546d2482b38e4aa0878ed6):


```
        exec-sql [0 args]
        CREATE TABLE cfs (a INT PRIMARY KEY, b STRING, c STRING, d STRING, FAMILY (b), FAMILY (c));
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:16:
        exec-sql [0 args]
        INSERT INTO cfs SELECT x, repeat('abc', 100), repeat('abc', 100) FROM generate_series(0, 3) AS x;
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:20:
        exec-sql [0 args]
        ALTER TABLE cfs SPLIT AT SELECT a FROM cfs;
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:24:
        exec-sql [0 args]
        -- Split the output files very small to catch output SSTs mid-row.
        SET CLUSTER SETTING bulkio.backup.file_size = '1';
        SET CLUSTER SETTING kv.bulk_sst.target_size = '1';
        SET CLUSTER SETTING bulkio.backup.merge_file_buffer_size = '1MiB';
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:31:
        exec-sql [0 args]
        BACKUP cfs TO 'nodelocal://1/foo';
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:35:
        exec-sql [0 args]
        CREATE DATABASE r1;
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:39:
        exec-sql [0 args]
        RESTORE cfs FROM 'nodelocal://1/foo' WITH into_db='r1';
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:43:
        query-sql [0 args]
        SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE orig.cfs];
        ----
        2
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:52: SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE r1.cfs];
        expected:
        2
        
        found:
        <nil>
    --- FAIL: TestBackupRestoreDataDriven/column-families (3.27s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #78069 ccl/backupccl: TestBackupRestoreDataDriven failed [C-test-failure O-robot T-bulkio branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupRestoreDataDriven [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4690908&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4690908&tab=artifacts#/) on master @ [834eaa0e83350486830867b5edd6e8809b52aa55](https://github.com/cockroachdb/cockroach/commits/834eaa0e83350486830867b5edd6e8809b52aa55):


```
        exec-sql [0 args]
        CREATE TABLE cfs (a INT PRIMARY KEY, b STRING, c STRING, d STRING, FAMILY (b), FAMILY (c));
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:16:
        exec-sql [0 args]
        INSERT INTO cfs SELECT x, repeat('abc', 100), repeat('abc', 100) FROM generate_series(0, 3) AS x;
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:20:
        exec-sql [0 args]
        ALTER TABLE cfs SPLIT AT SELECT a FROM cfs;
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:24:
        exec-sql [0 args]
        -- Split the output files very small to catch output SSTs mid-row.
        SET CLUSTER SETTING bulkio.backup.file_size = '1';
        SET CLUSTER SETTING kv.bulk_sst.target_size = '1';
        SET CLUSTER SETTING bulkio.backup.merge_file_buffer_size = '1MiB';
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:31:
        exec-sql [0 args]
        BACKUP cfs INTO 'nodelocal://1/foo';
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:35:
        exec-sql [0 args]
        CREATE DATABASE r1;
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:39:
        exec-sql [0 args]
        RESTORE cfs FROM LATEST IN 'nodelocal://1/foo' WITH into_db='r1';
        ----
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:43:
        query-sql [0 args]
        SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE orig.cfs];
        ----
        2
    backup_test.go:283: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3733/execroot/cockroach/bazel-out/k8-dbg/bin/pkg/ccl/backupccl/backupccl_test_/backupccl_test.runfiles/cockroach/pkg/ccl/backupccl/testdata/backup-restore/column-families:52: SELECT max(length(start_key)) FROM [SHOW RANGES FROM TABLE r1.cfs];
        expected:
        2
        
        found:
        <nil>
    --- FAIL: TestBackupRestoreDataDriven/column-families (6.87s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #78069 ccl/backupccl: TestBackupRestoreDataDriven failed [C-test-failure O-robot T-bulkio T-kv branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreDataDriven.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/backupccl/BUILD.bazel
+++ b/pkg/ccl/backupccl/BUILD.bazel
@@ -208,6 +208,7 @@ go_test(
         "//pkg/kv/bulk",
         "//pkg/kv/kvclient/kvcoord",
         "//pkg/kv/kvserver",
+        "//pkg/kv/kvserver/closedts",
         "//pkg/kv/kvserver/kvserverbase",
         "//pkg/kv/kvserver/protectedts",
         "//pkg/kv/kvserver/protectedts/ptpb",

--- a/pkg/ccl/backupccl/backup_test.go
+++ b/pkg/ccl/backupccl/backup_test.go
@@ -52,6 +52,7 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/kv"
 	"github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord"
 	"github.com/cockroachdb/cockroach/pkg/kv/kvserver"
+	"github.com/cockroachdb/cockroach/pkg/kv/kvserver/closedts"
 	"github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts"
 	"github.com/cockroachdb/cockroach/pkg/roachpb"
 	"github.com/cockroachdb/cockroach/pkg/security"
@@ -172,16 +173,23 @@ func (d *datadrivenTestState) addServer(
 	var cleanup func()
 	params := base.TestClusterArgs{}
 	params.ServerArgs.ExternalIODirConfig = ioConf
+	params.ServerArgs.Knobs = base.TestingKnobs{
+		JobsTestingKnobs: jobs.NewTestingKnobsWithShortIntervals(),
+	}
+
+	settings := cluster.MakeTestingClusterSettings()
+	closedts.TargetDuration.Override(context.Background(), &settings.SV, 10*time.Millisecond)
+	closedts.SideTransportCloseInterval.Override(context.Background(), &settings.SV, 10*time.Millisecond)
+
 	if tempCleanupFrequency != "" {
 		duration, err := time.ParseDuration(tempCleanupFrequency)
 		if err != nil {
 			return errors.New("unable to parse tempCleanupFrequency during server creation")
 		}
-		settings := cluster.MakeTestingClusterSettings()
 		sql.TempObjectCleanupInterval.Override(context.Background(), &settings.SV, duration)
 		sql.TempObjectWaitInterval.Override(context.Background(), &settings.SV, time.Millisecond)
-		params.ServerArgs.Settings = settings
 	}
+	params.ServerArgs.Settings = settings
 
 	clusterSize := singleNode
 

